pp_netlib/nx_label_comps.py

```diff
@@ -0,0 +1,20 @@
+import networkx as nx
```

Comment:
If this is a helper file for tests, it probably belongs in the `tests/` directory